local.py

```diff
@@ -4,6 +4,10 @@
 from server.response import *
 
 
+# def sleep(delay: float):
+#     pass
```

Comment:
clean up dead code or replace with comment?

Reply:
I left it here because it’s really useful while debugging. Should prob be controlled behind an environment variable tho. Keeps the output from taking forever when you’re just trying to test stuff